helm/flink-kubernetes-operator/conf/log4j-console.properties

```diff
@@ -60,3 +60,5 @@ appender.rolling.strategy.max = 10
 # Suppress the irrelevant (wrong) warnings from the Netty channel handler
 logger.netty.name = org.apache.flink.shaded.akka.org.jboss.netty.channel.DefaultChannelPipeline
 logger.netty.level = OFF
+
+monitorInterval = 30
```

Comment:
What's the interval? Seconds? Can we add this in a comment which also explains what this does?

Reply:
I think we should disable automatic reloading by default.